lib/app/sequence_base.rb

```diff
@@ -880,14 +880,14 @@ def fetch_all_bundled_resources(reply, reply_handler = nil)
           resources += bundle&.entry&.map { |entry| entry&.resource }
           next_bundle_link = bundle&.link&.find { |link| link.relation == 'next' }&.url
           reply_handler&.call(reply)
+
           break if next_bundle_link.blank?
 
           reply = @client.raw_read_url(next_bundle_link)
           error_message = "Could not resolve next bundle. #{next_bundle_link}"
           assert_response_ok(reply, error_message)
-          assert_valid_json(reply.body, error_message)
```

Comment:
Why don't we need this assertion anymore?

Reply:
If this body is invalid, @client.parse_reply would return nil
https://github.com/fhir-crucible/fhir_client/blob/42c675db614dd20812f617181b119af4a7d1b000/lib/fhir_client/client.rb#L315

Reply:
So now a server would pass if the next link in a bundle didn't return a body?